dist/lib/messages.d.ts

```diff
@@ -1,9 +1,33 @@
 import Request from "./request";
+declare type Attachment = {
+    data: ReadableStream;
+    filename: string;
+    contentType: string;
+    knownLength?: number;
+};
+declare type createData = {
+    to: string | string[];
```

Comment:
There are some missing fields compared to what was available in @types/mailgun-js. See https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b22f97ec443638166ac2f4d245c693ea4c495cd7/types/mailgun-js/index.d.ts#L85-L113

It's be great to have the common email headers, like `Sender`, `In-Reply-To`, etc.